content/efcore/getting-started/samples/generating-databases-and-data-models/generate-database/README.md

```diff
@@ -0,0 +1,9 @@
+# Generating from Code
+
+This code example demonstrates how to generate a database from a data model and print its resulting schema through `REPLService`.
+
+## Code Files:
+- [Book.cs](Book.cs)
+- [Program.cs](Program.cs)
+- [Reader.cs](Reader.cs)
+- [LibraryContext.cs](LibraryContext.cs)
```

Comment:
In other examples we have `Program.cs` as the first file and `LibraryContext.cs` as the second file so they both display in the console right away. Could we do that here too please?